packages/core/src/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -37,6 +38,19 @@ const Notification = ({ data, removeNotificationMessage }) => {
             </div>
             <div className='notification__text-container'>
                 <h4 className='notification__header'>{data.header}</h4>
+                {data.timeout && (
+                    <LinearProgress
+                        className='notification__timeout'
+                        timeout={data.timeout}
+                        action={data.action.onClick}
+                        render={remaining => (
+                            <Localize
+                                i18n_default_text='Auto update in {{ remaining }} seconds'
+                                values={{ remaining }}
+                            />
+                        )}
```

Comment:
Would this prop work to be passed in so that this text is not static?
Perhaps it's fine for now as it should be simple to move when needed.

Reply:
Yeah I think we can make it so that it comes from the notification config, but since this is the first of its name, I did not optimize it.